backend/src/routes/v0/project/__init__.py

```
@@ -109,6 +109,38 @@ async def get_project(
+    store: AsyncPostgresStore = Depends(get_store),
+):
+    """Update project name and/or description."""
+    service_context = ServiceContext(user_id=user.id, store=store)
+    try:
+        updated_project: Project = await service_context.project_service.update(
+            project_id,
+            {"name": project.name, "description": project.description},
+        )
+        return {"project": updated_project.model_dump(exclude_none=True)}
+    except ValueError as e:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=str(e))
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)